.github/workflows/set-date.yaml

```diff
@@ -0,0 +1,27 @@
+name: Set Date
+
+on:
+  schedule:
+    - cron: '* 0 * * *'
```

Comment:
Please add a comment into the action explaining the logic.

Is it currently just an unconditional daily change? The desire is to have it conditional on other changes in repo. I.e. while all collaboraters are on vacation and no one touches repo, automation should NOT change repo.

Reply:
OK, I will add a comment. Yes, that was @mck1117 's suggestion.
What are the downsides to this changing the repo without there being any changes? It will not trigger a build.

Reply:
why would it not trigger a build?
it's a git push, i would expect other actions to pick it up?

Reply:
From https://help.github.com/en/actions/reference/events-that-trigger-workflows#triggering-new-workflows-using-a-personal-access-token
"When you use the repository's GITHUB_TOKEN to perform tasks on behalf of the GitHub Actions app, events triggered by the GITHUB_TOKEN will not create a new workflow run."
This does in fact apply in our case.

Reply:
If we run it on schedule, but only change the date stamp if the repo has changed, The date could be behind by up to a day.

We can't really try to change it the way workflows are set up now, because there is no way to control which workflows run first, so some builds will not have the updated date and will lag behind by one build.